code/modules/unit_tests/modify_fantasy_variable.dm

```diff
@@ -1,21 +1,45 @@
 // Unit test to make sure that there are no duplicate keys when modify_fantasy_variable is called when applying fantasy bonuses.
 // Also to make sure the fantasy_modifications list is null when fantasy bonuses are removed.
+/datum/unit_test/modify_fantasy_variable
+
 /datum/unit_test/modify_fantasy_variable/Run()
+	var/list/applicable_types = subtypesof(/obj/item) - uncreatables
 
-	for(var/obj/item/path as anything in subtypesof(/obj/item))
```

Comment:
WTF, how old is this test??? This is 30 extra seconds of CI, it definitely needs to be bumped down in priority

Reply:
it was added two weeks ago